perma_web/perma/celery_tasks.py

```diff
@@ -1350,6 +1352,29 @@ def format_filesize(i):
         wacz_size = 0
 
     formatted_wacz_size = format_filesize(wacz_size)
+    if wacz_size:
+        # retrieve S3's md5 hash of file it has
+        remote_hash = default_storage.connection.Object(
+            bucket_name=default_storage.bucket_name,
+            key=os.path.join(settings.MEDIA_ROOT, link.warc_storage_file())
+        ).e_tag.strip('"')
+
+        # calculate the same thing for the warc inside the wacz
+        blocksize = 2 ** 20
```

Comment:
I admit to extreme laziness: I simply copied the block size from Ben's old code and hoped that it still worked. I'll go see if there's a record of why, or if I can find S3 docs explaining. I think the story is something like "we need to use the same block size that S3 does, when it is calculating e_tags", but I'll find out if that's true, and will add a comment.

Reply:
Best I can tell testing locally.... it doesn't matter. I can't find any info on why we did it one megabyte at a time, last time.

Maybe I'll add a comment that says, "use a blocksize of 1MB, because it's LIL tradition"